### PR TITLE
Move to a non-cgo SQLite implementation.

### PR DESCRIPTION
**Pull request by spetrovic77**
This will help avoid some compilation issues on MacOS and Windows.

### CODE REVIEW

File: examples/chat/sqlstore.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/ServiceWeaver/weaver"
 	_ "github.com/go-sql-driver/mysql"
```
Comment (rgrandl) [inline examples/chat/sqlstore.go:25]: Any reason we keep this dependency here?

File: examples/chat/sqlstore.go [hunk 2]
```diff
@@ -158,8 +158,8 @@ func (s *sqlStore) Init(ctx context.Context) error {
 		},
 	} {
 		query := q.query
-		if cfg.Driver == "sqlite3" {
-			// sqlite3 does not work with AUTO_INCREMENT specified on primary keys.
+		if cfg.Driver == "sqlite" {
+			// sqlite does not work with AUTO_INCREMENT specified on primary keys.
```
Comment (rgrandl) [inline examples/chat/sqlstore.go:162]: Is this still true?

**Review by rgrandl — APPROVED** [review]

**Review by johanbrandhorst — APPROVED** [review]
Thanks for this 😄

File: examples/chat/sqlstore.go (hunk 2 above)
Comment (spetrovic77) [reply examples/chat/sqlstore.go:162]: Yes, I believe it is.

File: examples/chat/sqlstore.go (hunk 1 above)
Comment (spetrovic77) [reply examples/chat/sqlstore.go:25]: For the GKE case, which uses the Cloud SQL MySQL instance.

**Comment by spetrovic77** [comment]
> 

You're welcome, thanks for bringing it to our attention.

**Pull request merged**